rotkehlchen/data_import/importers/binance.py

```diff
@@ -434,6 +550,72 @@ def process_entry(
         importer.add_history_events(write_cursor=write_cursor, history_events=[staking_event])
 
 
+class BinanceUSDMProgram(BinanceSingleEntry):
+    """All Binance USDM Program Single Events are processed here"""
+    ACCOUNT = 'USD-MFutures'  # specifying account because fee is ambiguous
+    AVAILABLE_OPERATIONS = (
+        'Fee',
+        'Funding Fee',
+        'Realized Profit and Loss',
+    )
+
+    def is_entry(self, requested_operation: str, account: str, change: int) -> bool:
+        if requested_operation in ('Fee', 'Funding Fee') and change == abs(change):
+            return False
+        return requested_operation in self.AVAILABLE_OPERATIONS and account == self.ACCOUNT
+
+    def _get_event(self, timestamp: Timestamp, data: BinanceCsvRow) -> HistoryEvent:
+        """
+        Process USDM Program Events. It includes
+        Fee, Funding Fee, and Realized Profit and Loss. May raise:
+        - KeyError
+        """
+        event_identifier = f'{EVENT_IDENTIFIER_PREFIX}{hash_csv_row(data)}'
+        timestamp_ms = ts_sec_to_ms(timestamp)
+        amount = abs(data['Change'])
+        asset: AssetWithOracles = data['Coin']
```

Comment:
All these can just go directly in the constructor. No need for temporary variables
```suggestion
```